packages/perseus/src/widgets/interactive-graphs/graphs/components/arrowhead.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import {useTransformContext, vec} from "mafs";
+import {pathBuilder} from "../../../../util/svg";
+
+type Props = {
+    x: number;
+    y: number;
+    rotate: number; // degrees counterclockwise from the positive x-axis
+};
+
+// We use the pathBuilder here to scale up the SVG path coordinates used
+// elsewhere for arrowheads. We scale in the path itself instead of using a CSS
+// transform because a transform would also scale up the stroke weight, and we
+// don't want that.
+const arrowPath = pathBuilder()
+    .move(-3, 4)
+    .curve(-2.75, 2.5, 0, 0.25, 0.75, 0)
+    .curve(0, -0.25, -2.75, -2.5, -3, -4)
+    .scale(1.4)
```

Comment:
Note that this also changes the scaling of the arrowheads on the axis graph, since the `Arrowhead` component is used for the axis arrows as well as the linear graph lines. Previously, the scale factor was `1.25`. The new scale matches the legacy graph backgrounds better AFAICT.